packages/angular-workspace/nimble-angular/README.md

```diff
@@ -10,7 +10,7 @@ NI-styled UI components for [Angular](https://angular.io) applications
 
 ## Getting started
 
-The steps to use components from Nimble Angular are similar to using components from any other Angular library. You can see the [Example Client App](/packages/angular-workspace/example-client-app) project for an example.
+The steps to use components from Nimble Angular are similar to using components from any other Angular library. You can see the [Example Client App](../example-client-app) project for an example.
```

Comment:
READMEs should use absolute relative paths: https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#documentation-policies

Can see that it worked in a slightly older package: https://www.npmjs.com/package/@ni/nimble-angular/v/30.9.1

Issue is that `directory` was added to the package.json but it doesn't serve a real purpose: https://github.com/ni/nimble/blob/main/packages/angular-workspace/nimble-angular/package.json#L12

Think we should just remove "directory" from all the `package.json` files so that links behave correctly on npmjs and github uis for rendering markdown.